asreviewcontrib/datatools/entrypoint.py

```diff
@@ -59,6 +60,43 @@ def execute(self, argv):
                     type=str,
                     help="Persistent identifier used for deduplication. Default: doi.",
                 )
+                dedup_parser.add_argument(
+                    "--drop_similar",
+                    action='store_true',
+                    help="Drop similar records.",
+                )
+                dedup_parser.add_argument(
+                    "--similarity",
+                    default=0.98,
+                    type=float,
+                    help="Similarity threshold for deduplication. Default: 0.98.",
+                )
+                dedup_parser.add_argument(
+                    "--skip_abstract",
+                    action='store_true',
+                    help="Use only title for deduplication.",
+                )
+                dedup_parser.add_argument(
+                    "--discard_stopwords",
+                    action='store_true',
+                    help="Discard stopwords for deduplication.",
+                )
+                dedup_parser.add_argument(
```

Comment:
I also find this name and help description a bit confusing. It's about the similarity technique you use, right?

Reply:
Yes, this is to enable the use of the `ratio()` function along with the `real_quick_ratio()` and `quick_ratio()`. But I don't think the user has to understand these details, only that it will use a more stricter check.